bin/node-gyp.js

```diff
@@ -126,7 +118,6 @@ process.on('uncaughtException', function (err) {
 })
 
 function errorMessage () {
-  // copied from npm's lib/util/error-handler.js
```

Comment:
This is sort of an attribution (although the current path in npm now appears to be lib/utils/error-handler.js), do maybe keep?